website/docs/discover.md

```diff
@@ -0,0 +1,176 @@
+and run Docker or podman on this Linux instance, then access them from your system.
+
+### Containers on Linux systems
+
+On Linux systems, Docker is running as a daemon and creates a socket so clients (including the docker cli) can access it through its API.
+The daemon receives commands from the clients, and executes them to build containers, store images, etc.
+
+Podman, on the other hand, is daemonless. This means that the clients (including the podman cli) access directly
+the system resources (containers, filesystem, etc), using a Podman library, to do these operations.
+
+Podman can also run on a Docker-compatibility mode. When doing this, you cannot run Docker and Podman simultaneously anymore.
+To run in this mode, a socket is created and a podman service is running, listening on this socket. This way, podman resources
+can be accessed in two ways. The first way is by connecting to the Podman socket (for CLIs which have been originally designed to work with Docker only)
+the exact same way the Docker socket is accessed. The second way is to use the podman cli (or the podman library), which accesses the resources directly.
+
+Podman desktop accesses the Docker daemon through the socket to work with Docker resources,
+and uses the podman cli to work with Podman resources.
+
+### Containers on non-Linux systems
+
+On Windows and macOS systems, a Linux system instance needs to be started inside a virtual machine and, on this Linux system, Podman
+needs to be started as a service, so it can be accessed from outside the virtual machine (you local system), through ssh.
+
+On these systems, Podman desktop uses the _remote_ version of the podman cli, which communicates with the serice inside the virtual machine through
+a ssh connection.
+
+## Installation
+
+Podman Desktop can be installed on macOS, Windows and Linux systems, with x64 or arm64 architectures. New versions are released regularly and binaries
+for these releases can be found in [the GitHub repository of the project](https://github.com/containers/podman-desktop). More detailed [instructions for the installation](https://podman-desktop.io/docs/installation) are available.
+
+The installation procedure consists of installing a binary and its dependent files in your system. Once done, you are ready to start Podman Desktop.
+
+## Onboarding
+
+When started for the first time, Podman Desktop starts an _Onboarding_ session. This session
+will guide you through installing resources and tools helpful when working with Podman Desktop.
+
+Podman Desktop checks for the following CLIs to be accessible on your system: `docker-compose`, `kubectl` and `podman`.
+If any of these CLIs are not accessible, you are proposed to choose which ones you want to install, and to start the Onboarding.
+
+The onboarding consists of a series of steps for each tool to install. For the `docker-compose` and `kubectl` CLIs,
```

Comment:
Sometimes onboarding is written using an upper ase, sometimes with a lowercase, no oref but need to pick one